CDP4_JSDK_Ecore_Modeling_Framework_Script.ps1

```diff
@@ -0,0 +1,29 @@
+﻿Set-ExecutionPolicy Unrestricted
+$srcFile = "C:\Users\kwojnowski\workspace-papyrus\cdp4emfmodel\model\CDP4.ecore"  # source file 
```

Comment:
this should should not be in the repos, works only for kamil

Reply:
Totally right. I forgot about this paths